initrd-trustee_plus/scripts/aa-client-service.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+. /lib/dracut-lib.sh
```

Comment:
In general, I'm wondering if we could avoid to build a new initrd and simply run this as a systemd service, it would simplify a lot the setup